### PR TITLE
Added territories to drop-down menu

### LINKED ISSUE #1659: Include jurisdictions in interactive visualization
should add at least DC, PR, ideally all of the territories that are not states...

### PR DESCRIPTION
**Pull request by ydhuang28**
Resolves #1659

Still need to find a solution for territories on the map.

### CODE REVIEW

File: visualization/scripts/convert_forecasts.py [hunk 1]
```diff
@@ -53,7 +53,8 @@ def reformat_forecasts(file_path, target, root, data):
               'Montana', 'Nebraska', 'Nevada', 'New Hampshire', 'New Jersey', 'New Mexico', 'New York',
               'North Carolina', 'North Dakota', 'Ohio', 'Oklahoma', 'Oregon', 'Pennsylvania', 'Rhode Island',
               'South Carolina', 'South Dakota', 'Tennessee', 'Texas', 'Utah', 'Vermont', 'Virginia', 'Washington',
-              'West Virginia', 'Wisconsin', 'Wyoming', 'District of Columbia']
+              'West Virginia', 'Wisconsin', 'Wyoming', 'District of Columbia',
+              'Puerto Rico', 'Virgin Islands', 'Guam', 'Northern Mariana Islands', 'American Samoa']
```
Comment (hannanabdul55) [inline visualization/scripts/convert_forecasts.py:57]: Why would we need these right now if we cannot visualize them yet? I see that we have not added these to the map as of now, right?

**Review by hannanabdul55 — APPROVED** [review]
Minor comment, but looks good otherwise!

File: visualization/scripts/convert_forecasts.py (hunk 1 above)
Comment (ydhuang28) [reply visualization/scripts/convert_forecasts.py:57]: No, not yet. But they can be selected through the drop-down menu, which is useful. The way that the map visualization works makes it hard to find a map with US state boundaries and all the territories, so this is a temporary solution while I try to find a working map.

**Comment by nickreich** [comment]
I'm happy to merge this without the map solution. maybe we could keep our eyes out for how other sites (both covid related and not) deal with this issue? maybe there is something that has a map inset somewhere?

**Comment by ydhuang28** [comment]
Agreed. Merging!

**Pull request merged**